superset/examples/birth_names.py

```diff
@@ -108,11 +108,24 @@ def load_birth_names(
         print(f"Creating table [{tbl_name}] reference")
         obj = TBL(table_name=tbl_name)
         db.session.add(obj)
-    obj.main_dttm_col = "ds"
+
+    _set_table_metadata(obj, database)
+    _add_table_metrics(obj)
+
+    db.session.commit()
+
+    slices, _ = create_slices(obj)
+    create_dashboard(slices)
+
+
+def _set_table_metadata(obj: "BaseDatasource", database: "Database") -> None:
```

Comment:
nit: `obj: "BaseDatasource"` is called `tbl: "BaseDatasource"` on `create_slices` we could name them equally maybe `datasource`?

Reply:
done